packages/@coorpacademy-components/locales/en/global.json

```diff
@@ -20,6 +20,7 @@
   "Create a new cohort": "Create a new cohort",
   "Delete": "Delete",
   "Draft": "Draft",
+  "answer_the_question": "Answer the question",
```

Comment:
"List of the questions" me semble plus correcte !

Reply:
penses y : à chaque fois c'est un group d'elements qui forment une réponse, ca n'a pas grand chose à avoir avec une liste de questions